tests/end-to-end/external-solution-integration/pkg/testkit/events.go

```diff
@@ -42,7 +42,7 @@ func NewEventSender(httpClient *http.Client, domain, application string) *EventS
 	return &EventSender{
 		httpClient: resilient.WrapHttpClient(httpClient),
 		domain:     domain,
-		ceClient:   ceClient,
+		ceClient:   http2.NewWrappedCloudEventClient(ceClient),
```

Comment:
(nit) I know you didn't introduce the import name `http2`, but can you rename it to something more meaningful? :)

Reply:
Done